draft-jholland-quic-multicast.md

```diff
@@ -64,15 +64,24 @@ This document does not define any multicast transport except server to client.
 
 # Multicast Session
 
-A multicast session (or just session) is a one-way communication stream by which a server can transport data to a client as part of the QUIC connection between the server and client.
+A multicast session (or just session) is a one-way network path that a server can use as an alternate path to send QUIC connection data to a client.
 
-Multicast sessions are additional network paths that can optionally be used by the server and client to transport application data concurrently with other network paths used by the connection.
+Multicast sessions are designed to leverage multicast IP and to be shared by many different connections simultaneously for unidirectional server-initiated data.
+Many servers can use the same session to send the same data to many clients.
```

Comment:
This isn't technically a new addition but I just noticed it, what kind of server do you mean when you speak of "many servers"? I assume its QUIC servers (processes)? The way its phrased now sounds a bit like it could describe ASM (many-to-many), which I am aware is not the intention.